src/lib/parts.ts

```diff
@@ -198,7 +209,7 @@ export class NodePart implements Part {
       this._commitTemplateResult(value);
     } else if (value instanceof Node) {
       this._commitNode(value);
-    } else if (Array.isArray(value) || value[Symbol.iterator]) {
+    } else if (isIterable(value)) {
```

Comment:
`isIterable` does more checks than what it's replacing. By this point in the `if` chain we know the value is not `null` and not a string.

Reply:
Fair point. Reverted.